kviewmodel/src/commonMain/kotlin/com/adeo/kviewmodel/KViewModel.kt

```diff
@@ -1,17 +1,29 @@
 package com.adeo.kviewmodel
 
+import kotlinx.coroutines.CoroutineExceptionHandler
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.cancel
+import kotlin.coroutines.CoroutineContext
+import kotlin.native.concurrent.ThreadLocal
 
 public abstract class KViewModel {
 
+    private val mainCoroutineExceptionHandler: CoroutineExceptionHandler? by lazy {
```

Comment:
By the way, I can optimize a little bit this code: 
`by lazy(LazyThreadSafetyMode.NONE) {`
Take me time :)

Reply:
Fixed